test/test_mps.py

```diff
@@ -3543,6 +3547,116 @@ def test_slice(self):
         mps_slice4 = mps_x[1, :].to('cpu')
         self.assertEqual(cpu_slice4, mps_slice4)
 
+    def test_slice_reshape_view_api_test_1(self):
+        x = torch.randn(4, 4)
+        x_mps = x.detach().clone().to("mps")
+
+        y = x[1]
+        y_mps = x_mps[1]
+        self.assertEqual(y, y_mps)
+
+        z = y.reshape(2, 2)
+        z_mps = y_mps.reshape(2, 2)
+        self.assertEqual(z, z_mps)
+        self.assertEqual(z.storage_offset(), z_mps.storage_offset())
+
+        r = z + 1
+        r_mps = z_mps + 1
+        self.assertEqual(r, r_mps)
+
+    def test_slice_reshape_view_api_test_2(self):
+        x = torch.randn(2, 4)
+        x_mps = x.detach().clone().to("mps")
+        ones = torch.ones(4)
+        ones_mps = torch.ones(4, device="mps")
+
+        print(x_mps)
```

Comment:
print?

Reply:
Thanks - this is some leftover debug code. Will update